File: app/src/main/java/com/example/test_store/Register/RegisterContract.java
package com.example.test_store.Register;


public interface RegisterContract {
    interface View{
        void onLoginClick(android.view.View v);
        void onRegisterClick(android.view.View v);
        void showToast(String message);
    }

    interface Presenter{
        void onRegisterClicked();
        void openLoginActivity();
    }
}

File: app/src/main/java/com/example/test_store/Register/RegisterContract.kt
package com.example.test_store.Register

import android.view.View

interface RegisterContract {
    interface View {
        fun onLoginClick(v: android.view.View?)
        fun onRegisterClick(v: android.view.View?)
        fun showToast(message: String?)
    }

    interface Presenter {
        fun onRegisterClicked()
        fun openLoginActivity()
    }
}